src/main/java/com/example/JavaProject/request/RegisterRequest.java

```diff
@@ -0,0 +1,4 @@
+package com.example.JavaProject.request;
+
+public class RegisterRequest {
```

Comment:
Are you planning on using RegisterRequest?
That class is empty

Reply:
I must have forgotten about it. It is gone now